Gemfile

```diff
@@ -42,6 +42,8 @@ gem 'rswag-ui'
 # uri parsing and generation
 gem 'addressable'
 
+gem 'descriptive_statistics', :require => 'descriptive_statistics/safe'
```

Comment:
I would like you to start leaving links to gem repos so they can be checked more easily. My assumption is this comes from [here?](https://github.com/thirtysixthspan/descriptive_statistics), in which case it hasn't been updated in over a year. Repo owner doesn't appear to be actively developing it anymore: https://github.com/thirtysixthspan/descriptive_statistics/issues/48#issuecomment-490995025

Reply:
On another thought, we should potentially do a check of all the gems to see which ones have become un-maintained, and list them for replacement later on

Reply:
> I would like you to start leaving links to gem repos so they can be checked more easily. 
No. See https://rubygems.org/gems/

Maintenance doesn't really matter for a library like this. Still, I found a more maintained version and am using that.